processor/k8sprocessor/kube/client.go

```diff
@@ -45,18 +45,26 @@ type WatchClient struct {
 	deleteQueue     []deleteRequest
 	stopCh          chan struct{}
 
-	Pods    map[string]*Pod
-	Rules   ExtractionRules
-	Filters Filters
+	Pods         map[string]*Pod
```

Comment:
Add a comment to explain what the key of the map is. I think it has a different meaning now, right?

Reply:
Previously the key was IP address, currently it can be either IP and UID. I'll put comment here